spring-cloud-starter-dataflow-server/src/test/java/org/springframework/cloud/dataflow/server/single/security/LocalServerSecurityWithOAuth2Tests.java

```diff
@@ -244,6 +245,7 @@ public void testAccessSecurityInfoUrlWithOAuth2AccessTokenPasswordGrant2Times()
 	}
 
 	@Test
+	@Ignore("Until we migrate from Spring Security OAuth")
```

Comment:
Do you mind explaining what the issue is/was and what do you mean by "Migrate from Spring Security OAuth"? 

Also, are we certain this is not an issue w/ the security changes recently made? Did this work prior to your changes?